### PR TITLE
Enable Integration tests procedure and add CONTRIBUTING MD file

### PR DESCRIPTION
**Pull request by nirarg**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
+# Contributing to the Cloud Azure Ops Ansible Collection
+ 
+We welcome community contributions to this collection.
+This guide provides clear instructions for contributors on how to set up their environment for contributing to the `cloud.azure_ops` Ansible Collection.
+If you find problems, please open an issue or create a PR against this collection repository.
+
+## Prerequisites
+
+Make sure you have the following installed on you Machine:
```
Comment (prabinovRedhat) [inline CONTRIBUTING.md:9]:
```suggestion
Make sure you have the following installed on your Machine:
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+   Prepare the Azure configuration file at `<root directory>/ansible_collections/cloud/azure_ops/tests/integration/cloud-config-azure.ini`. A template of this file is available in [the Ansible repository](https://github.com/ansible/ansible/blob/devel/test/lib/ansible_test/config/cloud-config-azure.ini.template). Populate the file with your appropriate credentials and resource group information.
+
+## Running tests
+
+To run the tests, change directory into azure-ops directory:
+```shell
+   cd <root directory>/ansible_collections/cloud/azure_ops
+```
+
+### Integration tests
```
Comment (prabinovRedhat) [inline CONTRIBUTING.md:51]: Maybe add some instructions on how to run the specific test?

Comment (prabinovRedhat) [reply CONTRIBUTING.md:51]: ansible-test integration <name of the specific test> --docker

Comment (prabinovRedhat) [reply CONTRIBUTING.md:51]: `ansible-test integration <name of the specific test> --docker`

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+We welcome community contributions to this collection.
+This guide provides clear instructions for contributors on how to set up their environment for contributing to the `cloud.azure_ops` Ansible Collection.
+If you find problems, please open an issue or create a PR against this collection repository.
+
+## Prerequisites
+
+Make sure you have the following installed on you Machine:
+
+1. Python3 and pip3
+2. Docker or Podman
```
Comment (anna-savina) [inline CONTRIBUTING.md:12]: I didn't succeeded to run tests without Docker install

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+3. Ansible
+   This can be installed by running `pip3 install ansible`
+
+## Environment Setup
+
+To set up your environment for contributing to the Cloud Azure Ops Ansible Collection, follow these steps:
+
+> **_NOTE:_** Make sure to replace `<root directory>` with the actual path where you want to clone the repositories and set up your environment.
```
Comment (anna-savina) [inline CONTRIBUTING.md:20]: Maybe it is better to replace `<root directory>` with `<working directory>`

Comment (anna-savina) [reply CONTRIBUTING.md:20]: and add:

And add to not privilege user permission to run docker if needed.
sudo usermod -a -G docker ${USER}
reboot

File: CONTRIBUTING.md (hunk 3 above)
Comment (nirarg) [reply CONTRIBUTING.md:12]: Did you have podman installed and it didn't work?

Comment (anna-savina) [reply CONTRIBUTING.md:12]: yes

File: README.md [hunk 5]
```diff
@@ -80,34 +80,10 @@ Once installed, you can reference the cloud.azure_ops collection content by its
...
-```
-
-There are also integration tests in the `molecule` directory which are meant to be run against an azure subscription.
-
-```shell
-    az login
-    molecule test
-```
-
-This collection is tested using GitHub Actions. To know more about CI, refer to [CI.md](CI.md).
```
Comment (alinabuzachis) [inline README.md:110]: I guess we should include the information regarding the CI testing.

File: CONTRIBUTING.md [hunk 6]
```diff
@@ -0,0 +1,73 @@
+# Contributing to the Cloud Azure Ops Ansible Collection
+ 
+We welcome community contributions to this collection.
+This guide provides clear instructions for contributors on how to set up their environment for contributing to the `cloud.azure_ops` Ansible Collection.
+If you find problems, please open an issue or create a PR against this collection repository.
```
Comment (alinabuzachis) [inline CONTRIBUTING.md:5]:
```suggestion
If you find problems, please open an issue or create a Pull Request (PR) against this collection repository.
```

File: CONTRIBUTING.md (hunk 3 above)
Comment (nirarg) [reply CONTRIBUTING.md:12]: Following to ansible documentation it should work
https://docs.ansible.com/ansible/latest/dev_guide/testing_running_locally.html#containers

File: CONTRIBUTING.md (hunk 1 above)
Comment (nirarg) [reply CONTRIBUTING.md]: Fixed

File: CONTRIBUTING.md (hunk 2 above)
Comment (nirarg) [reply CONTRIBUTING.md:44]: Added, thanks

File: CONTRIBUTING.md (hunk 4 above)
Comment (nirarg) [reply CONTRIBUTING.md]: Changed `<root directory>` to`<working directory>`
Added some links for additional info about how to configure

File: CONTRIBUTING.md [hunk 7]
```diff
@@ -0,0 +1,73 @@
...
+
+## Environment Setup
+
+To set up your environment for contributing to the Cloud Azure Ops Ansible Collection, follow these steps:
+
+> **_NOTE:_** Make sure to replace `<root directory>` with the actual path where you want to clone the repositories and set up your environment. 
+
+1. **Create Directories**:
+
+   Create the following directories in your home directory:
```
[the remark this answers is not in the document]
Comment (nirarg) [reply CONTRIBUTING.md]: Yes, you right, I removed this step

File: README.md (hunk 5 above)
Comment (nirarg) [reply README.md:110]: Yes, you right, I included this line back in the README

File: CONTRIBUTING.md (hunk 6 above)
Comment (nirarg) [reply CONTRIBUTING.md]: Done

File: CONTRIBUTING.md (hunk 3 above)
Comment (nirarg) [reply CONTRIBUTING.md]: Its seems in the Ansible docs that Podman should be supported
I guess you have local/configuration issue
I will try to run with podman and will let you know, but this is not related to this PR

File: CONTRIBUTING.md [hunk 8]
```diff
@@ -0,0 +1,72 @@
...
+3. Ansible
+   This can be installed by running `pip3 install ansible`
+
+## Environment Setup
+
+To set up your environment for contributing to the Cloud Azure Ops Ansible Collection, follow these steps:
+
+> **_NOTE:_** Make sure to replace `<working directory>` with the actual path where you want to clone the repositories and set up your environment. 
+
+1. **Clone This Repository**:
```
Comment (anna-savina) [inline CONTRIBUTING.md:24]:
```suggestion
1. **Clone Azure Roles Collection Repository**:
```

Comment (nirarg) [reply CONTRIBUTING.md]: I changed the descriptions in the clone steps.
Please review

**Comment by anna-savina** [comment]
lgtm
Thank you!

**Comment by prabinovRedhat** [comment]
lgtm, all tests passed

**Review by p3ck — APPROVED** [review]
lgtm

**Review by p3ck — APPROVED** [review]

**Comment by p3ck** [comment]
Looks like the changelog format isn't quite right..

`ERROR: Found 1 changelog issue(s) which need to be resolved:
ERROR: changelogs/fragments/20240416-integration.yml:0:0: section "bugfixes" list items must be type str not dict`

**Comment by nirarg** [comment]
> Looks like the changelog format isn't quite right..
> 
> `ERROR: Found 1 changelog issue(s) which need to be resolved: ERROR: changelogs/fragments/20240416-integration.yml:0:0: section "bugfixes" list items must be type str not dict`

@p3ck fixed this error, I think it ready for merge now
@alinabuzachis can you please review it again?

File: CONTRIBUTING.md [hunk 9]
```diff
@@ -0,0 +1,74 @@
...
+   ```shell
+   git clone https://github.com/ansible-collections/azure.git <working directory>/ansible_collections/azure/azcollection
+   ```
+
+3. ***Prepare Azure Configuration File***:  
+Prepare the Azure configuration file at `<working directory>/ansible_collections/cloud/azure_ops/tests/integration/cloud-config-azure.ini`. A template of this file is available in [the Ansible repository](https://github.com/ansible/ansible/blob/devel/test/lib/ansible_test/config/cloud-config-azure.ini.template). Populate the file with your appropriate credentials and resource group information.
+
+## Running tests
+
+To run the tests, change directory into azure-ops directory:
```
Comment (alinabuzachis) [inline CONTRIBUTING.md:41]:
```suggestion
To run the tests, change directory into azure_ops directory:
```

File: CONTRIBUTING.md [hunk 10]
```diff
@@ -0,0 +1,74 @@
...
+
+### Lint tests
+
+The project uses `ansible-lint` and `black`.
+To execute these tools, run:
+```shell
+  tox -e linters
+```
+
+
```
Comment (alinabuzachis) [inline CONTRIBUTING.md:74]:
```suggestion
```

File: CONTRIBUTING.md (hunk 9 above)
Comment (nirarg) [reply CONTRIBUTING.md]: Changed

File: CONTRIBUTING.md (hunk 10 above)
Comment (nirarg) [reply CONTRIBUTING.md]: Removed, thanks

**Review by p3ck — APPROVED** [review]

**Pull request merged**